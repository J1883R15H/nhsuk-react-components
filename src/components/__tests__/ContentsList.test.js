import React from 'react';
import { mount } from 'enzyme';
import ContentsList from '../contents-list';

describe('ContentsList', () => {
  it('item matches snapshot', () => {
    const mountedComponent = mount(
      <ContentsList.Item>Test Text</ContentsList.Item>
    );
    expect(mountedComponent).toMatchSnapshot();
    mountedComponent.unmount();
  });
  it('current item matches snapshot', () => {
    const mountedComponent = mount(
      <ContentsList.Item current>Test Text</ContentsList.Item>
    );
    expect(mountedComponent).toMatchSnapshot();
    mountedComponent.unmount();
  });
  it('container matches snapshot', () => {
    const mountedComponent = mount(
      <ContentsList>
        <ContentsList.Item>Test</ContentsList.Item>
        <ContentsList.Item current>Test Text</ContentsList.Item>
      </ContentsList>
    );
    expect(mountedComponent).toMatchSnapshot();
    mountedComponent.unmount();
  });
});
