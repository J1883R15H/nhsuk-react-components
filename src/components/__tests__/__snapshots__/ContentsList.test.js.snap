// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentsList container matches snapshot 1`] = `ReactWrapper {}`;

exports[`ContentsList current item matches snapshot 1`] = `ReactWrapper {}`;

exports[`ContentsList item matches snapshot 1`] = `ReactWrapper {}`;
